src/extension.ts

```diff
@@ -106,9 +106,36 @@ export function activate(context: ExtensionContext) {
             window.showMessage('Svelte language server restarted.');
         }
     }
+
+    function getLS() {
+        return ls;
+    }
+
+    addDidChangeTextDocumentListener(getLS);
+
```

Comment:
add `disposable` to context subscriptions `context.subscriptions.push(addDidChangeTextDocumentListener(getLS))` so that coc.nvim can handle when user disable this extension